src/lib/Timeline.js

```diff
@@ -606,23 +601,37 @@ export default class ReactCalendarTimeline extends Component {
     )
   }
 
-  selectItem = (item, clickType, e) => {
-    if (
-      this.state.selectedItem === item ||
-      (this.props.itemTouchSendsClick && clickType === 'touch')
-    ) {
-      if (item && this.props.onItemClick) {
-        const time = this.timeFromItemEvent(e)
-        this.props.onItemClick(item, e, time)
-      }
-    } else {
-      this.setState({ selectedItem: item })
-      if (item && this.props.onItemSelect) {
-        const time = this.timeFromItemEvent(e)
-        this.props.onItemSelect(item, e, time)
-      } else if (item === null && this.props.onItemDeselect) {
-        this.props.onItemDeselect(e) // this isnt in the docs. Is this function even used?
-      }
+  /* 
+    TODO: the itemTouchSendsClick and clickType logic here 
```

Comment:
I believe this is related to on some touch devices (specifically ios) touch doesn't fire click events unless they have certain attributes:

https://makandracards.com/makandra/34753-how-to-fix-ipad-does-not-trigger-click-event-on-some-elements

Don't worry about this for now.